generated/nidaqmx/_task_modules/in_stream.py

```diff
@@ -227,7 +227,7 @@ def avail_samp_per_chan(self):
         return val.value
 
     @property
-    def change_detect_overflowed(self):
+    def change_detect_has_overflowed(self):
```

Comment:
man these are hard... LV uses the shortened version, and C tends to use the phrases. I think I like the shortened version better. For example, we use `aux_power_error_chans_exist` instead of `aux_power_error_chans_do_exist` in Python and C. So I think the shortened one is more consistent, lets use `change_detect_overflowed`

Reply:
Reverted back to the older name